.devcontainer/Dockerfile

```diff
@@ -0,0 +1,10 @@
+ARG IMAGE=bullseye
```

Comment:
using bulleye is for any particular reason?

Reply:
Reading over things again, I realized I could simplify some things/ensure it's using the latest version of Python. I'm using the devcontainer images that Microsoft publishes to MCR.